### PR TITLE
Add LIAR dataset

### PR DESCRIPTION
**Pull request by hugoabonizio**
Add LIAR dataset from [“Liar, Liar Pants on Fire”: A New Benchmark Dataset for Fake News Detection](https://www.aclweb.org/anthology/P17-2067/).

### CODE REVIEW

File: datasets/liar/README.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+- unknown
+multilinguality:
+- monolingual
+size_categories:
+- 10K<n<100K
+source_datasets:
+- original
+task_categories:
+- text-classification
+task_ids: []
```
Comment (lhoestq) [inline datasets/liar/README.md:18]:
```suggestion
task_ids:
- text-classification-other-fake-news-detection
```

File: datasets/liar/liar.py [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+    VERSION = datasets.Version("1.1.0")
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "id": datasets.Value("string"),
+                    "label": datasets.Value("string"),
```
Comment (lhoestq) [inline datasets/liar/liar.py:63]: could you use the ClassLabel feature type here for the `label` field ?
for example
```
"label": datasets.Classlabel(names=["false", "half-true", etc.])
```

you'll need to regenerate the dataset_infos.json file after this change

**Review by lhoestq — COMMENTED** [review]
Thanks for adding this one !

I left a few comments. One is about the feature type of the `label` field
Let me know what you think :)

**Comment by hugoabonizio** [comment]
@lhoestq done! The failing testes don't seem to be related, it seems to be a connection issue, if I understand it correctly.

**Review by lhoestq — APPROVED** [review]
Nice thank you :)

**Comment by lhoestq** [comment]
merging since the CI is fixed on master

**Pull request merged**